### PR TITLE
Client.py and error no 35 fix

### PR DESCRIPTION
**Pull request by cpeng365**

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+        resp = None
+
+        try:
+            # Send handshake
+            logger.debug('sending "%s"' % handshake)
+            sock.sendall((urllib.parse.quote_plus(handshake)+'\n').encode())
+
+            # Look for the response
+            resp = self.__recv(sock)
```
Comment (mwalsh161) [inline client.py:74]: Might be worth checking that the response is the expected response.

File: client.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            print(resp)
```
Comment (mwalsh161) [inline client.py]: There should be no print statements anywhere in final implementation

File: client.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+            # Look for the response
+            resp = self.__recv(sock)
+            ip = resp[0]
+            port = resp[1]
+            logger.debug('received ["%s",%d]' % (ip, port))
+
+        finally:
+            self.__close_socket(sock)
+        return 'Client IP: %s, Port: %d' % (ip, port)
```
Comment (mwalsh161) [inline client.py]: Let’s return the values rather than a formatted string

**Review by mwalsh161 — CHANGES_REQUESTED** [review]
A few pretty minor changes. All print statements should be logger statements and all return values should be the actual values, not formatted strings.

File: client.py (hunk 1 above)
Comment (cpeng365) [reply client.py:74]: You mean checking if the response is "ack"?
I added a line "assert resp == 'ack', '%s does not exist' % module". Let me know if that works.

Comment (mwalsh161) [reply client.py:74]: Yep - except I would say a more general error; "Wan't able to get an acknowledgement from the server" (your error message makes an assumption that we don't need to make)

Comment (cpeng365) [reply client.py:74]: Got it! Just changed it.

**Review by mwalsh161 — APPROVED** [review]
Looks good!

**Pull request merged**